package.json

```
@@ -10,6 +10,7 @@
     "commit:all": "npm run commit:add && npm run commit",
     "readme:toc": "doctoc README.md --maxlevel 3 --title '## Table of Contents'",
     "test": "jest --colors",
+    "lint": "eslint \"**/*.js\"",
```

ESLint will automatically only lint JS files

```suggestion
    "lint": "eslint .",
```